one/one/main.cpp

```diff
@@ -0,0 +1,389 @@
+    }
+}
+
+void ThreeFourThree()
+{
+    while (true)
+    {
+        int n;
+        cin >> n;
+        if(n < 0)
+        {
+            cout << "Goodbye!" << endl;
+            break;
+        }
+        if(n % 5 != 0)
+        {
+            continue;
+        }
+        cout << n / 5;
+    }
+}
+
+/*
+    I really have no clue what this function does other an accumlate
+    2 ^ n bobs when n doles are odd...
+    ... just looked at answers apparently something called
+    russian peasant multiplication... never heard of it.
+ */
+/*
+void ThreeFiveOne()
+{
+    int accumulator;
+    while(true)
+    {
+        if(dole == 0) break; // end the loop when dole is 0
+        accumulator += ((dole % 2 == 1) ? bob : 0); //accumlate bob if dole
+                                                    // is odd
+        dole /= 2; // factor 2 out of dole
+        bob *= 2; // multiply bob by 2
+    }
+    cout << accumulator << endl; // printout accumulation
+}
+ */
+
+
+/*
+    This program always returns 1. OS would thing program errored
+    (nonzero return)
+ */
+/*
+void ThreeFiveTwo()
+{
+ #define o 1 // that is an oh not zero
+ 
+ int main()
+ {
+    return o;
+ }
+}
+ */
+
+/* Getting lazy, tired of rewriting source */
+
+/* 3.5.3 */
+/*
+    I really don't know, calculates squares of inverse values up to N.
+    Then within each iteration flip the sign of the accumulation up to i
+    times.
+    
+    The answer flew over my head too, something about a series
+    (calc 2 was a real struggle for me in uni)
+ */
+
+/*
+    Yes this compiles despite awkward forloop. It calculates factorials
+ */
+void FourOne()
+{
+    short number;
+    cout << "Enter a number...";
+    cin >> number;
+    
+    cout << "the factorial of " << number << " is ";
+    int accumulator = 1;
+    for(; number > 0; accumulator *= number--);
+    cout << accumulator << endl;
+}
+
+/*
+    Factorials cannot be calculated for negative numbers.
+ */
+void FourTwo()
+{
+    short number;
+    cout << "Enter a number...";
+    cin >> number;
+    
+    if(number < 0) {
+        cout << "cannot calcuate factorials of negative numbers" << endl;
+        return;
+    }
+    
+    cout << "the factorial of " << number << " is ";
+    int accumulator = 1;
+    for(; number > 0; accumulator *= number--);
+    cout << accumulator << endl;
+}
+
+/* 4.3 */
+/*
+    Factorials exponentiate pretty quickly, we will overflow int eventually
+ */
+
+void FourFour()
+{
+    short number;
+    cout << "Enter a number...";
+    cin >> number;
+    
+    if(number < 0) {
+        cout << "cannot calcuate factorials of negative numbers" << endl;
+        return;
+    }
+    
+    cout << "the factorial of " << number << " is ";
+    long long accumulator = 1;
+    for(; number > 0; accumulator *= number--);
+    cout << accumulator << endl;
+}
+
+/* 4.5 */
+/* it would be a big hardcoded switch statement... */
+/* a more interesting implementation (and intuitive)
+    would be using recursion
```

Comment:
Great idea: bonus points for a recursive factorial() function.
I know you functional programming people like recursion :)